src/webui/webapplication.cpp

```diff
@@ -117,6 +117,7 @@ QMap<QString, QMap<QString, WebApplication::Action> > WebApplication::initialize
     ADD_ACTION(command, decreasePrio);
     ADD_ACTION(command, topPrio);
     ADD_ACTION(command, bottomPrio);
+    ADD_ACTION(command, setAutoTMM);
```

Comment:
You should add an entry in wiki: https://github.com/qbittorrent/qBittorrent/wiki/WebUI-API-Documentation